twingate/datasource_groups.go

```diff
@@ -22,30 +28,74 @@ func datasourceGroupsRead(ctx context.Context, resourceData *schema.ResourceData
 	data := make([]interface{}, 0, len(groups))
 	for _, group := range groups {
 		data = append(data, map[string]interface{}{
-			"id":   group.ID.(string),
-			"name": string(group.Name),
+			"id":        group.ID.(string),
+			"name":      string(group.Name),
+			"type":      string(group.Type),
+			"is_active": bool(group.IsActive),
 		})
 	}
 
 	if err := resourceData.Set("groups", data); err != nil {
 		return diag.FromErr(err)
 	}
 
-	resourceData.SetId(groupName)
+	resourceData.SetId("groups-by-filter")
```

Comment:
I don't know enough about how datasources work, but does setting this to a static value:
* cause refresh problems?
* cause problems with multiple data sources?

Reply:
tested locally -  all good

also added test with two datasource